### DISCUSSION TITLE
The Pit [CHALLENGE MAP]

### DISCUSSION THREAD

**Discussion by ImTander** [nzp-team/nzportable#687] [Map Releases]
Hello! Welcome to my second custom map, The Pit! 

I wanted to make a simple/basic map that could be played smoothly across all platforms, and so I came up with a simple challenge map! Great for small/short play sessions!

How long can you survive in "The Pit", with zombies getting thrown at you from each side? 

Map Features : 
- Eight perks
- PaP
- The Mystery Box
- Ammo Mannequin 
- Buyable ending 
- Four player support

<img width="1920" height="1080" alt="1" src="https://github.com/user-attachments/assets/b9af60dc-96dc-47e1-b5b3-eff70e162a9a" />
<img width="1920" height="1080" alt="2" src="https://github.com/user-attachments/assets/5e15a8f1-2ea0-4896-8691-c7ec24fae510" />
<img width="1920" height="1080" alt="3" src="https://github.com/user-attachments/assets/2c72fb8e-e839-4eff-bbe0-087d4ae7efc7" />

# [DOWNLOAD](https://archive.org/download/the-pit_nzp/The%20Pit.zip)

Good Luck!

**Comment by ImTander** [comment]
**( UPDATED : 1/23/24 )**

- Added perk jingles
- Adjusted Double Tap perk machine model to Double Tap 1.0

**Comment by ImTander** [comment]
**( MAJOR UPDATE : 2/14/26 )**

- Added four more perks
- Double tap 1.0 changed to Double Tap 2.0
- Added "Ammo Mannequin"
- Added Buyable Ending
- Changes to skybox color

**Reply by ImTander** [reply]
**ADDITONAL UPDATE :** 
- Increased Ammo Mannequin price for challenge integrity
- Reverted Double Tap change for challenge integrity
- Changes to Mystery Box weapon pool

**Comment by nonMat** [comment]
This was quite fun! 
It's also really easy but I wouldn't say that's a bad thing!

**Reply by nonMat** [reply]
Yay for the changes

**Reply by ImTander** [reply]
Thank you for checking it out!
